### DISCUSSION TITLE
Webmate 3.x doesn't translate words in JS code.

### DISCUSSION THREAD

**Discussion by danardf** [WeblateOrg/weblate#7936] [General] [answered]
Hi.
For now we are using an old version of weblate (3.x) and the translation doesn't work for the words enclosed like this: **_("hello word")**
I'd like to know if there is a way to translate the words from JS code or we need to update Weblate to the latest version translating these words successfully?

Thanks for you help.

Franck.

**Comment by nijel** [comment, marked as answer]
Weblate doesn't parse JavaScript files, use xgettext to extract strings from it.

**Reply by danardf** [reply]
Thanks for your reply
Is there some example somewhere?

**Reply by danardf** [reply]
Ok I saw... 
Thanks I will try.
